src/external/adobe_coeff.c

```diff
@@ -273,8 +273,8 @@ static void dt_dcraw_adobe_coeff(const char *name, float cam_xyz[1][12])
     { "Leaf Credo 60", { 8035,435,-962,-6001,13872,2320,-1159,3065,5434 } },
     { "Leaf Credo 80", { 8035,435,-962,-6001,13872,2320,-1159,3065,5434 } },
     { "Leaf", { 8236,1746,-1314,-8251,15953,2428,-3673,5786,5771 } },
-    { "Leica M9 monochrom", { 6260,-1019,-470,-3730,11450,1930,-1409,2950,6210 } }, /* dummy */
-    { "Leica M9", { 6260,-1019,-470,-3730,11450,1930,-1409,2950,6210 } },
+//    { "Leica M9 monochrom", { 6260,-1019,-470,-3730,11450,1930,-1409,2950,6210 } }, /* dummy */
```

Comment:
either you add comments saying why this is commented out or you just remove the code.

Reply:
ok, remove it. M9 has a correct Matrix and the monochrome files don't need it.